templates/common/infra/bicep/core/storage/storage-account.bicep

```diff
@@ -2,10 +2,20 @@ param name string
 param location string = resourceGroup().location
 param tags object = {}
 
+@allowed([ 'Hot', 'Cool', 'Premium' ])
+param accessTier string = 'Hot'
 param allowBlobPublicAccess bool = false
+param allowCrossTenantReplication bool = true
```

Comment:
Should `allowCrossTenantReplication` be enabled by default? I could see this as something people don't want to enable cross-tenant access.

Reply:
It is true by default in ARM